python/lsst/sconsUtils/dependencies.py

```diff
@@ -76,13 +78,22 @@ def configure(packageName, versionString=None, eupsProduct=None, eupsProductPath
     state.env.libs = {"main": [], "python": [], "test": []}
     state.env.doxygen = {"tags": [], "includes": []}
     state.env['CPPPATH'] = []
-    state.env['LIBPATH'] = []
+
+    if 'SCONSUTILS_USE_CONDA_COMPILERS' in os.environ:
+        _conda_prefix = get_conda_prefix()
+        state.env['LIBPATH'] = ["%s/lib" % _conda_prefix]
+    else:
+        state.env['LIBPATH'] = []
 
     # XCPPPATH is a new variable defined by sconsUtils - it's like CPPPATH,
     # but the headers found there aren't treated as dependencies.  This can
     # make scons a lot faster.
     state.env['XCPPPATH'] = []
 
+    if 'SCONSUTILS_USE_CONDA_COMPILERS' in os.environ:
+        _conda_prefix = get_conda_prefix()
+        state.env.Append(XCPPPATH=["%s/include" % _conda_prefix])
+
```

Comment:
I think these are okay. This obviates some of the need to use the .config files we have for third parties in the simple case. @timj what do you think here?

Reply:
Aren't the .cfg files still needed in order to know which libraries you need to link against?

Reply:
ah right, I guess this only obviates certain things (ndarray?) and I forgot he was using remap for the time being.

Reply:
So these hooks simply let scons find the right compilers and insert proper linking paths for most things. The `.cfg` files are stored separately and use the already established APIs for telling scons how to find the libraries it needs. By using this layer of abstraction, one can avoid changes in the dependencies/env forcing changes in this repo.